src/canon/Physics.jl

```diff
@@ -157,7 +157,7 @@ end)
 )
 
 @docapode("Poiseuille Density"
```

Comment:
@lukem12345 @quffaro This link appears to be dead and there's no Wikipedia page on this specific topic. Can you provide a fresh link?

Reply:
This is the old link: https://en.wikipedia.org/wiki/Hagen-poiseuille_density

Reply:
https://en.wikipedia.org/wiki/Hagen%E2%80%93Poiseuille_equation